<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabImportExport</class>
 <widget class="QWidget" name="TabImportExport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="grpUpload">
     <property name="title">
      <string>Upload</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Dein Name:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="edtName">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="placeholderText">
         <string>Anonym</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="5">
       <widget class="QTableWidget" name="tblExport">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string/>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Kategorie</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Lektion</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Umfang</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Autor</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCommandLinkButton" name="btnUpload">
        <property name="text">
         <string>Hochladen</string>
        </property>
        <property name="icon">
         <iconset resource="../img.qrc">
          <normaloff>:/img/img/svn-commit.png</normaloff>:/img/img/svn-commit.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="grpDownload">
     <property name="title">
      <string>Download</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1">
       <widget class="QCommandLinkButton" name="btnDownload">
        <property name="text">
         <string>Herunterladen</string>
        </property>
        <property name="icon">
         <iconset resource="../img.qrc">
          <normaloff>:/img/img/svn-update.png</normaloff>:/img/img/svn-update.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCommandLinkButton" name="btnRefresh">
        <property name="text">
         <string>Aktualisieren</string>
        </property>
        <property name="icon">
         <iconset resource="../img.qrc">
          <normaloff>:/img/img/Wiederholen.png</normaloff>:/img/img/Wiederholen.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" rowspan="3">
       <widget class="QTableWidget" name="tblImport">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string/>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Kategorie</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Lektion</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Umfang</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Autor</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../img.qrc"/>
 </resources>
 <connections/>
</ui>
